src/main/java/de/samply/share/broker/filter/AuthenticationFilter.java

```diff
@@ -33,7 +35,8 @@
 public class AuthenticationFilter implements ContainerRequestFilter {
```

Comment:
Please split this filter into one for the JWT based authentication as before and one for the monitoring. This filter is just way to complex. And please add tests for both filters.